src/coreComponents/linearAlgebra/interfaces/hypre/HyprePreconditioner.hpp

```diff
@@ -127,16 +140,31 @@ class HyprePreconditioner final : public PreconditionerBase< HypreInterface >
   LinearSolverParameters m_parameters;
 
   /// Pointer to the Hypre implementation
-  HYPRE_Solver m_precond;
+  HYPRE_Solver m_precond = nullptr;
 
   /// Pointer to the auxillary preconditioner used in MGR
-  HYPRE_Solver aux_precond;
+  HYPRE_Solver aux_precond = nullptr;
```

Comment:
Same comment as for petsc, let's initialize in constructor

Reply:
Done.